### PR TITLE
Pin lib_xua to 4.1.0 (and changelog updates)

### PR DESCRIPTION
**Pull request by xross**

### CODE REVIEW

**Review by danielpieczko — COMMENTED** [review]
I noticed that we forgot to change `shared\version.h` in the last release, so just a reminder when you are updating version numbers.

**Comment by xross** [comment]
Thanks!

**Comment by xross** [comment]
I raised an issue regarding a test for this https://github.com/xmos/sw_usb_audio/issues/242

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,13 +1,48 @@
...
 =======================
 
-UNRELEASED
-----------
+8.1.0
+-----
 
+  * ADDED:     2AMi18o18mssaax build config (MIDI, SPDIF Rx/Tx & ADAT Rx/Tx)
+  * ADDED:     2AMi16o16xxxaax build config (ADAT Rx/Tx)
+  * ADDED:     2AMi18o8_mxxxxx build config (MIDI)
```
Comment (danielpieczko) [inline CHANGELOG.rst:9]: Should be 8 inputs instead of 18, and no underscore in the config name

**Review by danielpieczko — APPROVED** [review]

File: CHANGELOG.rst (hunk 1 above)
Comment (xross) [reply CHANGELOG.rst:9]: Thanks. I'll update spec document too (which also has this mistake)

**Pull request merged**